packages/kuma-gui/src/app/application/components/route-view/RouteView.vue

```diff
@@ -27,10 +27,11 @@
       :src="uri(sources, '/me/:route', {
         route: props.name,
       })"
+      @change="resolve"
       v-slot="{ data: me }"
     >
       <slot
-        v-if="me && submit"
+        v-if="me && submit && redirected"
```

Comment:
This is the extra template guard I reused to make sure we don't show anything until we've set our default query params. This is only necessary during CI for some reason, probably due to the single tick used in the `watch` as a consequence of the `async/await`